frontends/concrete-python/concrete/fhe/mlir/context.py

```diff
@@ -2442,6 +2536,15 @@ def tlu(self, resulting_type: ConversionType, on: Conversion, table: Sequence[in
         assert on.bit_width <= MAXIMUM_TLU_BIT_WIDTH
 
         table = list(table)
+
+        if all(value == table[0] for value in table[1:]):
```

Comment:
It's a table lookup, with constant table content, the value will be known in any case. Also, the result will be encrypted because of zeros and optional addition :slightly_smiling_face: